holmes/plugins/toolsets/grafana/toolset_grafana.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict, List
+from urllib.parse import urlencode, urljoin
+from holmes.core.tools import Tool, ToolParameter
+from holmes.plugins.toolsets.grafana.base_grafana_toolset import BaseGrafanaToolset
+import requests
+import logging
+
+
+class ListAndBuildGrafanaDashboardURLs(Tool):
```

Comment:
I think @aantn also implemented this
Are you two synced on this?

Reply:
I just checked, no dashboard tools currently in project